src/userWallet.ts

```diff
@@ -240,6 +242,79 @@ export abstract class UserWallet {
     return usdValue
   }
 
+  generate2fa = () => {
+    const { secret, uri, qr } = generateSecret({
+      name: yamlConfig.name,
+      account: this.user.phone,
+    })
+    /*
+    { secret: 'XDQXYCP5AC6FA32FQXDGJSPBIDYNKK5W',
```

Comment:
Why is the comment being committed?

Reply:
We have a few comments like this across the codebase where we have an example of the return value if it's not immediately clear.

Reply:
My comment
> Then this should be done via a type or a unit test. That would communicate the same but without the risk of getting out of sync

Was referring to this. We shouldn't have issues that refer to Adding Tests separately. Test code must be written together with (potentially even before) the production code and merged together.

Reply:
Ah okay, gotcha